### ISSUE TITLE
Publish latest v0.25.0 to pypi

### ISSUE THREAD

**Issue by liyakun** [tensorflow/federated#2917]
**Is your feature request related to a problem? Please describe.**
The latest release v0.25.0 is not on pypi yet.

**Describe the solution you'd like**
Publish the latest release v0.25.0 to pypi.

**Comment by michaelreneer** [comment]
There was a WORKSPACE dependency issue publishing the v0.25.0 release to PyPI. We intend to publish a Python package this week. I'm curious to know more about your use case and if a release this week will solve your issues.

**Comment by liyakun** [comment]
@michaelreneer thanks for your information. I got the following error while installing `grpcio` with python 3.10 

```bash
        File "/home/admin/.cache/pypoetry/virtualenvs/financial-fl-zn838s7N-py3.10/lib/python3.10/site-packages/setuptools/_distutils/unixccompiler.py", line 176, in _compile
            raise CompileError(msg)
        distutils.errors.CompileError: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
        
        [end of output]
    
    note: This error originates from a subprocess, and is likely not a problem with pip.
  error: legacy-install-failure
  
  × Encountered error while trying to install package.
  ╰─> grpcio

```

The `grpcio` package seems needed by `tensorflow`, `tensorboard` and `tensorflow-federated` (v0.24.0). 

After switching from python 3.10 to 3.9, the error was resolved. At the beginning, I thought the issue might be resolve by v0.25.0 of `tensorflow-federated`, therefore, I created the ticket to be able to try the new version.

**Comment by michaelreneer** [comment]
Glad you fixed your issue! Also we just released v0.26.0.
